### PR TITLE
added missing translation keys

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
@Soullivaneuh travis failed, can you please have a look?

File: Resources/translations/SonataClassificationBundle.fr.xliff [hunk 1]
```diff
@@ -154,6 +154,18 @@
                 <source>classification.tree_mode</source>
                 <target>Arbre</target>
             </trans-unit>
+            <trans-unit id="breadcrumb.link_category_tree">
+                <source>breadcrumb.link_category_tree</source>
+                <target>breadcrumb.link_category_tree</target>
+            </trans-unit>
+            <trans-unit id="active">
+                <source>active</source>
+                <target>active</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataClassificationBundle.fr.xliff]: Actif

File: Resources/translations/SonataClassificationBundle.fr.xliff [hunk 2]
```diff
@@ -154,6 +154,18 @@
...
+            </trans-unit>
+            <trans-unit id="disabled">
+                <source>disabled</source>
+                <target>disabled</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataClassificationBundle.fr.xliff]: Désactivé

File: Resources/translations/SonataClassificationBundle.fr.xliff [hunk 3]
```diff
@@ -154,6 +154,18 @@
                 <source>classification.tree_mode</source>
                 <target>Arbre</target>
             </trans-unit>
+            <trans-unit id="breadcrumb.link_category_tree">
+                <source>breadcrumb.link_category_tree</source>
+                <target>breadcrumb.link_category_tree</target>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataClassificationBundle.fr.xliff]: Catégorie

**Comment by Soullivaneuh** [comment]
> @Soullivaneuh travis failed, can you please have a look?

PHPUnit issue, again. Wait a minute I'll fix that.

**Comment by Soullivaneuh** [comment]
Should be ok with fd8bcd88c512ff8303678ba9c925c74ab9cc8934.

Please rebase.

File: Resources/translations/SonataClassificationBundle.fr.xliff (hunk 1 above)
Comment (OskarStark) [inline Resources/translations/SonataClassificationBundle.fr.xliff]: ``Actif`` or ``activ``?

**Pull request merged**